recipe/meta.yaml

```diff
@@ -48,3 +48,5 @@ extra:
   recipe-maintainers:
     - mvanniekerk
```

Comment:
I think this is suppose to be, @mariusvniekerk . Would you mind correcting this as well?

xref: https://github.com/conda-forge/staged-recipes/pull/2834#discussion_r116907980